yang/ietf-l3vpn-ntw.yang

```diff
@@ -753,15 +753,28 @@ module ietf-l3vpn-ntw {
                   description
                     "An identifier of an active VPN instance profile.";
                 }
+                leaf bearer-reference {
+                    if-feature "vpn-common:bearer-reference";
+                    type string;
+                    description
+                      "This is an internal reference for the service
+                      provider.";
+                  }
                 uses vpn-common:service-status;
                 container connection {
                   description
                     "Encapsulation types.";
-                  leaf encapsulation-type {
-                    type identityref {
-                      base vpn-common:encapsulation-type;
+                  leaf connection-type {
+                    type enumeration {
+                        enum LOCAL {
+                          description
+                            "Local termination of the VPN network access.";
```

Comment:
Thanks Samier for updating the description. 

I still don't get this. What is a local/remote termination?